pages/test_analytics/ruby_collectors.md.erb

```diff
@@ -113,13 +124,15 @@ This would appear like so:
 
 This is particularly useful for tests that generate a lot of span data such as system/feature tests. You can find all _annotations_ under _Span Timeline_ at the bottom of every test execution page.
 
-## Troubleshooting `allow_any_instance_of` errors
+## Tagging duplicate test executions with a prefix/suffix
 
-If you're using RSpec and seeing errors related to `allow_any_instance_of`, which look like this:
+For builds that execute the same test multiple times it's possible to tag each test execution with a prefix/suffix describing the test environment. This is useful when running a test suite against multiple versions of Ruby or Rails. The prefix/suffix is set using these environment variables:
 
-```ruby
-Failure/Error: allow_any_instance_of(Object).to receive(:sleep)
-       Using `any_instance` to stub a method (sleep) that has been defined on a prepended module (Buildkite::TestCollector::Object::CustomObjectSleep) is not supported.
 ```
+BUILDKITE_ANALYTICS_EXECUTION_NAME_PREFIX
+BUILDKITE_ANALYTICS_EXECUTION_NAME_SUFFIX
+```
+
+When viewing a _test_ every _execution_ is displayed including its corresponding prefix/suffix. For example:
 
-you can fix them by being more specific in your stubbing, which is [better practice](https://relishapp.com/rspec/rspec-mocks/v/3-11/docs/working-with-legacy-code/any-instance), by replacing `allow_any_instance_of(Object).to receive(:sleep)` with `allow_any_instance_of(TheClassUnderTest).to receive(:sleep)`.
+<%= image "execution-prefix-suffix.png", width: 2086/2, height: 294/2, alt: "Screenshot of test executions including a prefix and suffix." %>
```

Comment:
```suggestion
<%= image "execution-prefix-suffix.png", width: 2048/2, height: 400/2, alt: "Screenshot of test executions including a prefix and suffix." %>
```
Hi @swebb just one last minor thing: if we could fix the image dimensions I think we're ready to go.

Reply:
Thanks @olyism.